pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/AvgAggregationFunction.java

```diff
@@ -65,6 +65,44 @@ public void aggregate(int length, AggregationResultHolder resultHolder, Object..
   }
 
   /**
+   * Performs 'avg' aggregation on the input array.
+   * Returns {@value #DEFAULT_VALUE} if the input array is empty.
+   *
+   * While the interface allows for variable number of valueArrays, we do not support
+   * multiple columns within one aggregation function right now.
+   *
+   * {@inheritDoc}
+   *
+   * @param length
+   * @param resultHolder
+   * @param valueArrayArray
+   */
+  @Override
+  public void aggregateMV(int length, AggregationResultHolder resultHolder, Object... valueArrayArray) {
```

Comment:
like sum all the values for a field, this can help reduce number of rows in dimension.
instead of do 
A | 1
A | 2
A | 3
I can do 
A | [1,2,3]

This will improve my filtering perf and save storage.